.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
```

Comment:
What does this do? Enable alerts?

Reply:
[Dependabot](https://docs.github.com/en/code-security/getting-started/dependabot-quickstart-guide) will open automatic pull requests whenever the following packages receive updates:

- "sphinx-scylladb-theme"
- "sphinx-multiversion-scylla"

Previously, patch updates were automatically installed, but we’ve now changed the process so each project can control when to update. This automation simplifies that control.